cuqi/experimental/mcmc/_pcn.py

```diff
@@ -5,13 +5,15 @@
 
 class pCNNew(SamplerNew):  # Refactor to Proposal-based sampler?
 
+    _STATE_KEYS = SamplerNew._STATE_KEYS + ['scale', 'current_loglike_logd']
+
     def __init__(self, target, scale=1.0, **kwargs):
 
         super().__init__(target, **kwargs)
 
         self.scale = scale
         self.current_point = self.initial_point
-        self.current_loglike_eval = self._loglikelihood(self.current_point)
+        self.current_loglike_logd = self._loglikelihood(self.current_point)
```

Comment:
suggest changing this variable to `current_likelihood_logd` (changing all occurrences of the variable) to be consistent with `current_target_logd`

Reply:
Done